lib/qesapdeployment.pm

```diff
@@ -472,7 +514,8 @@ sub qesap_execute {
     foreach (qw(cmd logname)) { croak "Missing mandatory $_ argument" unless $args{$_}; }
 
     my $verbose = $args{verbose} ? "--verbose" : "";
-    $args{cmd_options} ||= '';
+    $args{cmd_options} //= '';
```

Comment:
from `||=` to `//=` is to have syntax coherent across all the file but also because `//=` also handle `undef`